os/android/platformdefines.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+#include "platformdefines.h"
+
+#ifdef USE_VK
+#include <drm/drm_fourcc.h>
+#endif
```

Comment:
I think there would be more include instructions and function definitions in this file when necessary, so uses separated ifdef for include and function.